### ISSUE TITLE
 invalid escape sequence: `\U`

### ISSUE THREAD

**Issue by alexandrev** [eris-ltd/eris-cli#414]
It seems like there is a problem when you are trying to deploy a contract on Windows (client side) and it seems related to the way the software is handling the slashes and backslashes in the paths. I'm getting this error:

```
PS C:\Users\alexa\.eris\apps\idi> eris contracts  test --dir=C:/Users/alexa/.eris/apps/idi
panic: regexp: Compile(`eris_data_C:\Users\alexa\.eris\apps\idi_88fcbf03_1`): error parsing regexp: invalid escape sequence: `\U`

goroutine 1 [running]:
regexp.MustCompile(0xc082135dc0, 0x32, 0x5)
        c:/go/src/regexp/regexp.go:221 +0x176
github.com/eris-ltd/eris-cli/util.ParseContainers(0xc082135dc0, 0x32, 0xc081feaf01, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/util/container_stats.go:293 +0x216
github.com/eris-ltd/eris-cli/perform.ContainerExists(0xc08207d000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:625 +0x5a
github.com/eris-ltd/eris-cli/perform.DockerCreateData(0xc08207d000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:38 +0x14a
github.com/eris-ltd/eris-cli/chains.setupChain(0xc0820ec000, 0xa4b438, 0x3, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:244 +0x2be
github.com/eris-ltd/eris-cli/chains.NewChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:44 +0x4c1
github.com/eris-ltd/eris-cli/chains.ThrowAwayChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:92 +0x3c8
github.com/eris-ltd/eris-cli/contracts.bootThrowAwayChain(0xc08221a800, 0x1d, 0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:302 +0xa6
github.com/eris-ltd/eris-cli/contracts.BootServicesAndChain(0xc0820ec000, 0xc082238000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:85 +0x469
github.com/eris-ltd/eris-cli/contracts.RunPackage(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:37 +0x42e
github.com/eris-ltd/eris-cli/commands.ContractsTest(0xd40ce0, 0xc0821003d0, 0x0, 0x1)
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/contracts.go:149 +0x114
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).execute(0xd40ce0, 0xc082100380, 0x1, 0x1, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:530 +0x870
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0xd41e60, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:613 +0x59a
github.com/eris-ltd/eris-cli/commands.Execute()
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/eris.go:66 +0x39
main.main()
        D:/Go/src/github.com/eris-ltd/eris-cli/cmd/eris/main.go:8 +0x1b

goroutine 5 [select]:
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.readLoop()
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:116 +0x202
created by github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.init.1
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:28 +0x32

goroutine 6 [syscall]:
os/signal.loop()
        c:/go/src/os/signal/signal_unix.go:22 +0x1f
created by os/signal.init.1
        c:/go/src/os/signal/signal_unix.go:28 +0x3e

goroutine 18 [IO wait]:
net.runtime_pollWait(0x1295be0, 0x72, 0xc082143750)
        c:/go/src/runtime/netpoll.go:157 +0x67
net.(*pollDesc).Wait(0xc082021070, 0x72, 0x0, 0x0)
        c:/go/src/net/fd_poll_runtime.go:73 +0x41
net.(*ioSrv).ExecIO(0xc08202a1c0, 0xc082020f60, 0xa47b10, 0x7, 0xb4b450, 0x101d, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:182 +0x177
net.(*netFD).Read(0xc082020f00, 0xc08224a000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:463 +0x17e
net.(*conn).Read(0xc082148000, 0xc08224a000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/net.go:172 +0xeb
crypto/tls.(*block).readFromUntil(0xc082146870, 0x14a0000, 0xc082148000, 0x5, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:455 +0xd3
crypto/tls.(*Conn).readRecord(0xc08214c000, 0xb4bb17, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:540 +0x2d8
crypto/tls.(*Conn).Read(0xc08214c000, 0xc08215c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:901 +0x16e
net/http.noteEOFReader.Read(0x14a4468, 0xc08214c000, 0xc08210eb58, 0xc08215c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/net/http/transport.go:1370 +0x6e
net/http.(*noteEOFReader).Read(0xc0822044a0, 0xc08215c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        <autogenerated>:126 +0xd7
bufio.(*Reader).fill(0xc0821fd3e0)
        c:/go/src/bufio/bufio.go:97 +0x1f0
bufio.(*Reader).Peek(0xc0821fd3e0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
        c:/go/src/bufio/bufio.go:132 +0xd3
net/http.(*persistConn).readLoop(0xc08210eb00)
        c:/go/src/net/http/transport.go:876 +0xfe
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:685 +0xc7f

goroutine 19 [select]:
net/http.(*persistConn).writeLoop(0xc08210eb00)
        c:/go/src/net/http/transport.go:1009 +0x413
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:686 +0xca4
```

And I've seen the following StackOveflow thread that is seems related: http://stackoverflow.com/questions/13737370/cmake-error-invalid-escape-sequence-u

Versions: 
```
PS C:\Users\alexa\.eris\apps\idi> eris version
Eris CLI Version: 0.11.0
```

**Comment by compleatang** [comment]
Thanks for the bug report. 

We'll have a fix this weekend. We missed something in the eris:pm container creation but it should be an easy fix.

**Comment by alexandrev** [comment]
Thanks for the quick response. If I could be more helpful to do the testing or so, please tell me :)

**Comment by compleatang** [comment]
Indeed, if you could test the fix when we push it that would be very helpful. I should hope we can have that done by tomorrow if you are able to test then it would be much appreciated. Windows CI is still not fully finalized it at this point.

Was everything else working OK during the setup sequence? Getting chains running, etc.?

**Comment by compleatang** [comment]
@alexandrev am working on some OSX CI fixes and a few other things before merging this into develop, but if you want to test feel free (the work I'm doing shouldn't affect)

```
eris update --branch winConFix
```

or, after #415 merges into develop:

```
eris update --branch develop
```

 Note, currently using the update command requires go+git to be installed on your host box.

**Comment by alexandrev** [comment]
I'm not sure if I run all the commands but this is the output and it seems everything is working in the same way, so same error.

```
Eris CLI Version: 0.11.0
PS C:\Users\alexa\.eris\apps> eris update --branch winConFix
PS C:\Users\alexa\.eris\apps> eris contracts  test --dir=C:/Users/alexa/.eris/apps/idi
panic: regexp: Compile(`eris_data_C:\Users\alexa\.eris\apps_d12d37b4_1`): error parsing regexp: invalid escape sequence: `\U`

goroutine 1 [running]:
regexp.MustCompile(0xc0822568a0, 0x2e, 0x5)
        c:/go/src/regexp/regexp.go:221 +0x176
github.com/eris-ltd/eris-cli/util.ParseContainers(0xc0822568a0, 0x2e, 0xc081fed501, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/util/container_stats.go:293 +0x216
github.com/eris-ltd/eris-cli/perform.ContainerExists(0xc08207d000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:625 +0x5a
github.com/eris-ltd/eris-cli/perform.DockerCreateData(0xc08207d000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:38 +0x14a
github.com/eris-ltd/eris-cli/chains.setupChain(0xc0820ec000, 0xa4b438, 0x3, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:244 +0x2be
github.com/eris-ltd/eris-cli/chains.NewChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:44 +0x4c1
github.com/eris-ltd/eris-cli/chains.ThrowAwayChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:92 +0x3c8
github.com/eris-ltd/eris-cli/contracts.bootThrowAwayChain(0xc082218800, 0x19, 0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:302 +0xa6
github.com/eris-ltd/eris-cli/contracts.BootServicesAndChain(0xc0820ec000, 0xc082236000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:85 +0x469
github.com/eris-ltd/eris-cli/contracts.RunPackage(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:37 +0x42e
github.com/eris-ltd/eris-cli/commands.ContractsTest(0xd40ce0, 0xc0821003d0, 0x0, 0x1)
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/contracts.go:149 +0x114
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).execute(0xd40ce0, 0xc082100380, 0x1, 0x1, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:530 +0x870
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0xd41e60, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:613 +0x59a
github.com/eris-ltd/eris-cli/commands.Execute()
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/eris.go:66 +0x39
main.main()
        D:/Go/src/github.com/eris-ltd/eris-cli/cmd/eris/main.go:8 +0x1b

goroutine 5 [select]:
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.readLoop()
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:116 +0x202
created by github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.init.1
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:28 +0x32

goroutine 6 [syscall]:
os/signal.loop()
        c:/go/src/os/signal/signal_unix.go:22 +0x1f
created by os/signal.init.1
        c:/go/src/os/signal/signal_unix.go:28 +0x3e
```

**Comment by compleatang** [comment]
Shoot. OK. We've got our WindowsCI service back in action so that will help going forward but I'll keep investigating this issue.

**Comment by compleatang** [comment]
@alexandrev if you have a moment, can you rerun the `contracts test` command with the `--debug` flag? It will help me track down where this odd rendering of the appName is getting introduced.

**Comment by alexandrev** [comment]
Here you are:

```
PS C:\Users\alexa\.eris\apps\idi> eris update --branch winConFix
PS C:\Users\alexa\.eris\apps\idi> eris contracts  test --dir=C:/Users/alexa/.eris/apps/idi --debug
Connecting to the Docker Client via =>  tcp://192.168.99.100:2376:C:\Users\alexa\.docker\machine\machines\default
Checking Details via Docker-Machine =>  eris
Connecting to the Docker Client via =>  tcp://192.168.99.100:2376:C:\Users\alexa\.docker\machine\machines\default
Connecting to the Docker Client via TLS.
        URL =>                  tcp://192.168.99.100:2376
        Docker Cert Path =>     C:\Users\alexa\.docker\machine\machines\default
Connected over TLS.
Successfully connected to Docker daemon.
Setting IPFS Host
Set ERIS_IPFS_HOST to =>        http://192.168.99.100
Welcome! Say the Marmots. Running App package.
        with Host Path =>       C:/Users/alexa/.eris/apps/idi:C:\Users\alexa\.eris\apps\idi
The marmots could not read that package.json. Will use defaults.
Package testType =>             epm
        deployType =>           epm
        testTask =>             default
        deployTask =>           default
Setting App Type. Task =>       test
        ChainType =>            mint
        ChainName =>
        App Type =>             epm
No chain was given, booting a throwaway chain.
Making a ThrowAwayChain =>      C:\Users\alexa\.eris\apps\idi_75a8b2d2:C:\Users\alexa\.eris\chains\default
Starting Setup for ChnID =>     C:\Users\alexa\.eris\apps\idi_75a8b2d2
Asking Docker Client for the Existing Containers. Quiet? false
Loading Data Definition =>      C:\Users\alexa\.eris\apps\idi_75a8b2d2:1
Creating Data Container for =>  eris_data_C:\Users\alexa\.eris\apps\idi_75a8b2d2_1
Parsing Containers =>           eris_data_C:\Users\alexa\.eris\apps\idi_75a8b2d2_1:true
panic: regexp: Compile(`eris_data_C:\Users\alexa\.eris\apps\idi_75a8b2d2_1`): error parsing regexp: invalid escape sequence: `\U`

goroutine 1 [running]:
regexp.MustCompile(0xc0822569c0, 0x32, 0x5)
        c:/go/src/regexp/regexp.go:221 +0x176
github.com/eris-ltd/eris-cli/util.ParseContainers(0xc0822569c0, 0x32, 0xc08206a001, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/util/container_stats.go:293 +0x216
github.com/eris-ltd/eris-cli/perform.ContainerExists(0xc0821cdf00, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:625 +0x5a
github.com/eris-ltd/eris-cli/perform.DockerCreateData(0xc0821cdf00, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:38 +0x14a
github.com/eris-ltd/eris-cli/chains.setupChain(0xc0820ec000, 0xa4b438, 0x3, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:244 +0x2be
github.com/eris-ltd/eris-cli/chains.NewChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:44 +0x4c1
github.com/eris-ltd/eris-cli/chains.ThrowAwayChain(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:92 +0x3c8
github.com/eris-ltd/eris-cli/contracts.bootThrowAwayChain(0xc082151340, 0x1d, 0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:302 +0xa6
github.com/eris-ltd/eris-cli/contracts.BootServicesAndChain(0xc0820ec000, 0xc082248000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:85 +0x469
github.com/eris-ltd/eris-cli/contracts.RunPackage(0xc0820ec000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:37 +0x42e
github.com/eris-ltd/eris-cli/commands.ContractsTest(0xd40ce0, 0xc0820eeb80, 0x0, 0x2)
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/contracts.go:149 +0x114
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).execute(0xd40ce0, 0xc0820eea00, 0x2, 0x2, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:530 +0x870
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0xd41e60, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:613 +0x59a
github.com/eris-ltd/eris-cli/commands.Execute()
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/eris.go:66 +0x39
main.main()
        D:/Go/src/github.com/eris-ltd/eris-cli/cmd/eris/main.go:8 +0x1b

goroutine 5 [select]:
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.readLoop()
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:116 +0x202
created by github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.init.1
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:28 +0x32

goroutine 6 [syscall]:
os/signal.loop()
        c:/go/src/os/signal/signal_unix.go:22 +0x1f
created by os/signal.init.1
        c:/go/src/os/signal/signal_unix.go:28 +0x3e

goroutine 33 [IO wait]:
net.runtime_pollWait(0x13b5be8, 0x72, 0xc0822da120)
        c:/go/src/runtime/netpoll.go:157 +0x67
net.(*pollDesc).Wait(0xc082021070, 0x72, 0x0, 0x0)
        c:/go/src/net/fd_poll_runtime.go:73 +0x41
net.(*ioSrv).ExecIO(0xc08202a1c0, 0xc082020f60, 0xa47b10, 0x7, 0xb4b450, 0x1d, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:182 +0x177
net.(*netFD).Read(0xc082020f00, 0xc0822dc000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:463 +0x17e
net.(*conn).Read(0xc082154000, 0xc0822dc000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/net.go:172 +0xeb
crypto/tls.(*block).readFromUntil(0xc082152090, 0x34a0000, 0xc082154000, 0x5, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:455 +0xd3
crypto/tls.(*Conn).readRecord(0xc082158000, 0xb4bb17, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:540 +0x2d8
crypto/tls.(*Conn).Read(0xc082158000, 0xc082168000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:901 +0x16e
net/http.noteEOFReader.Read(0x34e47a0, 0xc082158000, 0xc08210f188, 0xc082168000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/net/http/transport.go:1370 +0x6e
net/http.(*noteEOFReader).Read(0xc082223aa0, 0xc082168000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        <autogenerated>:126 +0xd7
bufio.(*Reader).fill(0xc08222a060)
        c:/go/src/bufio/bufio.go:97 +0x1f0
bufio.(*Reader).Peek(0xc08222a060, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
        c:/go/src/bufio/bufio.go:132 +0xd3
net/http.(*persistConn).readLoop(0xc08210f130)
        c:/go/src/net/http/transport.go:876 +0xfe
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:685 +0xc7f

goroutine 34 [select]:
net/http.(*persistConn).writeLoop(0xc08210f130)
        c:/go/src/net/http/transport.go:1009 +0x413
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:686 +0xca4
```

**Comment by compleatang** [comment]
awesome thanks. this is definitely a path issue. am currently vetting these calls now. hope to have this fixed today. thanks, again, for your patience.

**Comment by alexandrev** [comment]
I'm facing the same issue. It seems we change things but we are stuck at the same step: 

```
PS D:\Projects\GitHub\eris-cli> eris version
Eris CLI Version: 0.11.0
PS D:\Projects\GitHub\eris-cli> eris contracts  test --dir=C:/Users/alexa/.eris/apps/idi --debug
Connecting to the Docker Client via =>  tcp://192.168.99.100:2376:C:\Users\alexa\.docker\machine\machines\default
Checking Details via Docker-Machine =>  eris
Connecting to the Docker Client via =>  tcp://192.168.99.100:2376:C:\Users\alexa\.docker\machine\machines\default
Connecting to the Docker Client via TLS.
        URL =>                  tcp://192.168.99.100:2376
        Docker Cert Path =>     C:\Users\alexa\.docker\machine\machines\default
Connected over TLS.
Successfully connected to Docker daemon.
Setting IPFS Host
Set ERIS_IPFS_HOST to =>        http://192.168.99.100
Welcome! Say the Marmots. Running App package.
        with Host Path =>       C:/Users/alexa/.eris/apps/idi:D:\Projects\GitHub\eris-cli
The marmots could not read that package.json. Will use defaults.
Package testType =>             epm
        deployType =>           epm
        testTask =>             default
        deployTask =>           default
Setting App Type. Task =>       test
        ChainType =>            mint
        ChainName =>
        App Type =>             epm
No chain was given, booting a throwaway chain.
Making a ThrowAwayChain =>      D:\Projects\GitHub\eris-cli_9e735122:C:\Users\alexa\.eris\chains\default
Starting Setup for ChnID =>     D:\Projects\GitHub\eris-cli_9e735122
Asking Docker Client for the Existing Containers. Quiet? false
Loading Data Definition =>      D:\Projects\GitHub\eris-cli_9e735122:1
Creating Data Container for =>  eris_data_D:\Projects\GitHub\eris-cli_9e735122_1
Parsing Containers =>           eris_data_D:\Projects\GitHub\eris-cli_9e735122_1:true
panic: regexp: Compile(`eris_data_D:\Projects\GitHub\eris-cli_9e735122_1`): error parsing regexp: invalid character clas
s range: `\Pr`

goroutine 1 [running]:
regexp.MustCompile(0xc082205f20, 0x30, 0x5)
        c:/go/src/regexp/regexp.go:221 +0x176
github.com/eris-ltd/eris-cli/util.ParseContainers(0xc082205f20, 0x30, 0xc082068001, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, .
..)
        D:/Go/src/github.com/eris-ltd/eris-cli/util/container_stats.go:293 +0x216
github.com/eris-ltd/eris-cli/perform.ContainerExists(0xc0821aee00, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:625 +0x5a
github.com/eris-ltd/eris-cli/perform.DockerCreateData(0xc0821aee00, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/perform/docker_run.go:38 +0x14a
github.com/eris-ltd/eris-cli/chains.setupChain(0xc0820f2000, 0xa4b438, 0x3, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:244 +0x2be
github.com/eris-ltd/eris-cli/chains.NewChain(0xc0820f2000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:44 +0x4c1
github.com/eris-ltd/eris-cli/chains.ThrowAwayChain(0xc0820f2000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/chains/operate.go:92 +0x3c8
github.com/eris-ltd/eris-cli/contracts.bootThrowAwayChain(0xc082218800, 0x1b, 0xc0820f2000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:302 +0xa6
github.com/eris-ltd/eris-cli/contracts.BootServicesAndChain(0xc0820f2000, 0xc082236000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:85 +0x469
github.com/eris-ltd/eris-cli/contracts.RunPackage(0xc0820f2000, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/contracts/operate.go:37 +0x42e
github.com/eris-ltd/eris-cli/commands.ContractsTest(0xd40ce0, 0xc0820f0d20, 0x0, 0x2)
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/contracts.go:149 +0x114
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).execute(0xd40ce0, 0xc0820f0ba0, 0x2
, 0x2, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:530 +0x870
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0xd41e60, 0x0, 0x0)
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/spf13/cobra/command.go:613 +0x59a
github.com/eris-ltd/eris-cli/commands.Execute()
        D:/Go/src/github.com/eris-ltd/eris-cli/commands/eris.go:66 +0x39
main.main()
        D:/Go/src/github.com/eris-ltd/eris-cli/cmd/eris/main.go:8 +0x1b

goroutine 5 [select]:
github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.readLoop()
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:116 +0x
202
created by github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log.init.1
        D:/Go/src/github.com/eris-ltd/eris-cli/Godeps/_workspace/src/github.com/eris-ltd/common/go/log/logger.go:28 +0x3
2

goroutine 6 [syscall]:
os/signal.loop()
        c:/go/src/os/signal/signal_unix.go:22 +0x1f
created by os/signal.init.1
        c:/go/src/os/signal/signal_unix.go:28 +0x3e

goroutine 18 [IO wait]:
net.runtime_pollWait(0x1575bd0, 0x72, 0xc082103a30)
        c:/go/src/runtime/netpoll.go:157 +0x67
net.(*pollDesc).Wait(0xc08201f1f0, 0x72, 0x0, 0x0)
        c:/go/src/net/fd_poll_runtime.go:73 +0x41
net.(*ioSrv).ExecIO(0xc0820281c0, 0xc08201f0e0, 0xa47b10, 0x7, 0xb4b450, 0x101d, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:182 +0x177
net.(*netFD).Read(0xc08201f080, 0xc08224a000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/fd_windows.go:463 +0x17e
net.(*conn).Read(0xc082152000, 0xc08224a000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
        c:/go/src/net/net.go:172 +0xeb
crypto/tls.(*block).readFromUntil(0xc0821508d0, 0x3050000, 0xc082152000, 0x5, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:455 +0xd3
crypto/tls.(*Conn).readRecord(0xc082156000, 0xb4bb17, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:540 +0x2d8
crypto/tls.(*Conn).Read(0xc082156000, 0xc082166000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/crypto/tls/conn.go:901 +0x16e
net/http.noteEOFReader.Read(0x3054568, 0xc082156000, 0xc082119658, 0xc082166000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        c:/go/src/net/http/transport.go:1370 +0x6e
net/http.(*noteEOFReader).Read(0xc08220a300, 0xc082166000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        <autogenerated>:126 +0xd7
bufio.(*Reader).fill(0xc082201980)
        c:/go/src/bufio/bufio.go:97 +0x1f0
bufio.(*Reader).Peek(0xc082201980, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
        c:/go/src/bufio/bufio.go:132 +0xd3
net/http.(*persistConn).readLoop(0xc082119600)
        c:/go/src/net/http/transport.go:876 +0xfe
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:685 +0xc7f

goroutine 19 [select]:
net/http.(*persistConn).writeLoop(0xc082119600)
        c:/go/src/net/http/transport.go:1009 +0x413
created by net/http.(*Transport).dialConn
        c:/go/src/net/http/transport.go:686 +0xca4
```

**Comment by compleatang** [comment]
am closing this as it appears we have the paths mostly fixed. feel free to log a new issue if we didn't fix something but finalizing the windows tests for fully greeen windows is on the slate for 0.11.1 (another week or two)
